include/nbl/video/ISwapchain.h

```diff
@@ -57,6 +57,8 @@ class NBL_API ISwapchain : public core::IReferenceCounted, public IBackendObject
             return { m_images->begin(), m_images->end() };
         }
 
+        inline video::ISurface::E_SURFACE_TRANSFORM_FLAGS getSurfaceTransform() const { return m_params.preTransform; }
```

Comment:
It actually doesn't give surface current transform UNLESS we always create swapchain with surface current transform using `vkGetPhysicalDeviceSurfaceCapabilitiesKHR` in vulkan,
So agree to name this to `getPreTransform`?
it has a logical name "pre" in it because it is done "prior" to presentation engine presenting the image.
and yes most of the time we want to set this to surface current transform